scripts/periodic

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+PATH="/home/ubuntu/grakn:$PATH"
+
+while true; do
+        graql console -e 'match $x isa pokemon; get;';
```

Comment:
This is not really periodic right? It's more a continuous and repeated query with no breaks?

Reply:
Yes, I guess so! It's always querying. I should rename it.